forward/fwd.go

```diff
@@ -190,18 +210,6 @@ func (f *httpForwarder) serveHTTP(w http.ResponseWriter, req *http.Request, ctx
 		ctx.log.Infof("Round trip: %v, code: %v, duration: %v",
 			req.URL, response.StatusCode, time.Now().UTC().Sub(start))
 	}
-
-	defer response.Body.Close()
-
-	if err != nil {
-		ctx.log.Errorf("Error copying upstream response Body: %v", err)
-		ctx.errHandler.ServeHTTP(w, req, err)
-		return
-	}
-
-	if written != 0 {
```

Comment:
I did the same a year ago in https://github.com/containous/oxy/commit/183212964e13e7b8afe01a08b193d04300554a68
But I had to [rollback](https://github.com/containous/oxy/commit/b57d6706e9ff606343c596940b60df7f90012d29) it to fix an empty response issue in traefik https://github.com/containous/traefik/pull/476
Any idea on this @tcolgate ?